pymemcache/test/test_integration.py

```diff
@@ -26,9 +26,7 @@
     MemcacheClientError
 )
 from pymemcache.serde import (
-    get_python_memcache_serializer,
-    python_memcache_serializer,
-    python_memcache_deserializer
+    PythonMemcachePickleSerde, python_memcache_pickle_serde
```

Comment:
Let's keep the "one name per line" format we had before.

Reply:
Sure. It reminds me that I meant to suggest maybe using `black`.

I feel like the parts of my brain which pay attention to formatting minutiae have been mostly "turned off" since I started using it.
But definitely, I'll fix to be consistent with what's here.